index.html

```diff
@@ -0,0 +1,108 @@
+    <link rel="shortcut icon" type="image/x-icon" href="https://cpwebassets.codepen.io/assets/favicon/favicon-aec34940fbc1a6e787974dcd360f2c6b63348d4b1f4e06c77743096d55480f33.ico">
+    <link rel="mask-icon" type="image/x-icon" href="https://cpwebassets.codepen.io/assets/favicon/logo-pin-8f3771b1072e3c38bd662872f6b673a722f4b3ca2421637d5596661b4e2132cc.svg" color="#111">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+
+<body>
+  <div class="container">
+    <header id="header">
+        <div id="logo">
+            <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="">
+          </div> 
+          <div class="nav-bar">
+            <nav id="nav-bar">
+                <ul>
+                    <li><a class="nav-link" href="#features">Features</a></li>
+                    <li> <a class="nav-link" href="#how-it-works">How It Works</a></li>
+                    <li><a class="nav-link" href="#pricing">Pricing</a></li>
+                </ul>
+
+            </nav>
+          </div>
+    </header>
+    
+    <div class="second-header">
+      <h2>
+        Handcrafted, home-made masterpieces
+      </h2>
+
+    
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+
+        <input name="email" id="email" type="email" placeholder="Enter your email address" required="">
+        <input id="submit" type="submit" value="Get Started" class="btn">
+    </form>
+  </div>
+    <section id="features">
+      <div class="grid">
+        <div class="icon">
+          <i class='fa fa-3x fa-fire'></i>
+        </div>
+        <div class="desc">
+          <h2>Premium Materials</h2>
+          <p>Our trombones use the shiniest brass which is sourced locally.
+             This will increase the longevity of your purchase.</p>
+        </div>
+      </div>
+      <div class="grid">
+        <div class="icon">
+          <i class='fa fa-3x fa-truck'></i>
+        </div>
+        <div class="desc">
+          <h2>Fast Shipping</h2>
+          <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+        </div>
+      </div>
+      <div class="grid">
+        <div class="icon">
+          <i class='fa fa-3x fa-truck'></i>
+        </div>
+        <div class="desc">
+          <h2>Quality Assurance</h2>
+          <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+        </div>
+      </div>
+
+    </section>
+    <section id="how-it-works">
+      <div class="grid">
+        <div class="icon">
+          <i>
+           
+          </i>
+        </div>
+        
+      </div>
+      <div class="grid">
+        <div class="icon">
+          <i ></i>
+        </div>
+      </div>
+
+    </section>
```

Comment:
Section lacks heading. Consider using h1 - h6 elements to add an identifying heading to all sections. Or don't use section tag in such situation.

Reply:
Please consider using h1 - h6 elements to add an identifying heading to all sections. Or you also can replace section tag in such situation with a div tag.